coalib/parsing/Globbing.py

```diff
@@ -29,6 +29,24 @@ def _end_of_set_index(string, start_index):
     return closing_index
 
 
+def glob_escape(string):
```

Comment:
Anything you like.
As Makman said, apparently he uses string everywhere. So thats ok too

On Thu, Mar 17, 2016 at 7:49 PM, Adhityaa Chandrasekar <
notifications@github.com> wrote:

> In coalib/parsing/Globbing.py
> <https://github.com/coala-analyzer/coala/pull/1644#discussion_r56510568>:
>
> > @@ -29,6 +29,24 @@ def _end_of_set_index(string, start_index):
> >      return closing_index
> >
> >
> > +def glob_escape(string):
>
> Does input_string sound better? We can't use just input either...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1644/files/5f92726abba97e9fc0733ca1d8b86bde85b0f011#r56510568>
>